lesson_21/homework_21.py

```diff
@@ -37,3 +37,81 @@
 
     1. Обробіть відпові
 """
+import requests
+
+BASE_URL = "https://petstore.swagger.io/v2"
+
+def get_all_pets():
+    url = f"{BASE_URL}/pet/findByStatus"
+    params = {'status': 'available,pending,sold'}
+    response = requests.get(url, params=params)
+    if response.status_code == 200:
+        all_pets = response.json()
+        print("Список усіх доступних тварин:")
+        for pet in all_pets:
+            name = pet.get('name', 'Без імені')
+            print(f"ID: {pet['id']}, Ім'я: {name}, Статус: {pet['status']}")
+    else:
+        print(f"Помилка отримання списку всіх тварин: {response.status_code} - {response.text}")
+
+def get_pets_by_status(statuses):
+    url = f"{BASE_URL}/pet/findByStatus"
+    params = {'status': ','.join(statuses)}
+    response = requests.get(url, params=params)
+    if response.status_code == 200:
+        pets = response.json()
+        print(f"Список тварин зі статусами {statuses}:")
```

Comment:
```suggestion
        print(f"Список тварин зі статусами {statuses}:")
```
от прінт це непогано, але як тоді далі тестувати?

Reply:
дякую за зауваженя, виправила
тепер функції повертають результати, і їх можна використовувати для подальшого тестування.

Reply:
https://github.com/alex-pancho/aqa_py_290424/pull/144/commits/7944c46d72ac640248f5e0b651b3a2f0be13f930